config/prod.secret.exs

```diff
@@ -0,0 +1,19 @@
+use Mix.Config
+
+# In this file, we keep production configuration that
+# you'll likely want to automate and keep away from
+# your version control system.
```

Comment:
Do we need this file to be in git? The description/comment seems to indicate that we should add it to gitignore.